maven/bnd-baseline-maven-plugin/src/main/java/aQute/bnd/maven/baseline/plugin/BaselineMojo.java

```diff
@@ -61,10 +61,17 @@
 	@Parameter(readonly = true, required = false)
 	private Base					base;
 
+    @Parameter(defaultValue = "false", readonly = true)
```

Comment:
Why is this parameter readonly? That makes it impossible to use, because it will always be set to `false`.